src/site/stages/build/plugins/create-header-footer.js

```diff
@@ -51,31 +18,19 @@ function createHeaderFooterData(buildOptions) {
       megaMenuData,
     };
 
-    const serialized = JSON.stringify(headerFooter, null, 4);
-
-    const drupalMenu = DRUPALS.PREFIXED_ENVIRONMENTS.has(buildOptions.buildtype)
-      ? replaceWithDrupalLinks(headerFooter, files)
-      : headerFooter;
```

Comment:
`PREFIXED_ENVIRONMENTS` is empty and has been for a long time so this code was always a noop.